### ISSUE TITLE
PytestDeprecationWarning: direct construction of Function has been deprecated

### ISSUE THREAD

**Issue by skewty** [pytest-dev/pytest-asyncio#147]
pytest version 5.4.1

```
=============================== warnings summary ===============================
/home/scott/Projects/combox2/venv37/lib/python3.7/site-packages/pytest_asyncio/plugin.py:39
/home/scott/Projects/combox2/venv37/lib/python3.7/site-packages/pytest_asyncio/plugin.py:39
  /home/scott/Projects/combox2/venv37/lib/python3.7/site-packages/pytest_asyncio/plugin.py:39: PytestDeprecationWarning: direct construction of Function has been deprecated, please use Function.from_parent
    item = pytest.Function(name, parent=collector)

/home/scott/Projects/combox2/venv37/lib/python3.7/site-packages/pytest_asyncio/plugin.py:45
/home/scott/Projects/combox2/venv37/lib/python3.7/site-packages/pytest_asyncio/plugin.py:45
  /home/scott/Projects/combox2/venv37/lib/python3.7/site-packages/pytest_asyncio/plugin.py:45: PytestDeprecationWarning: direct construction of Function has been deprecated, please use Function.from_parent
    item = pytest.Function(name, parent=collector)  # To reload keywords.

-- Docs: https://docs.pytest.org/en/latest/warnings.html
======================== 1 passed, 4 warnings in 0.09s =========================
```

**Comment by simonfagerholm** [comment]
Duplicate of #141

**Comment by simonfagerholm** [comment]
Fixed in #142
